src/pages/ChatPage/components/ReportChatModal.tsx

```
@@ -0,0 +1,196 @@
+        <div className="border-border mx-4 w-full max-w-sm rounded-2xl border bg-white p-6 text-center shadow-2xl">
+          <CheckCircle2 className="mx-auto mb-4 h-16 w-16 text-success" />
+          <h3 className="text-foreground mb-2 text-xl font-semibold">
+            {t("chat.report.success.title")}
+          </h3>
+          <p className="mb-6 text-muted-foreground">
+            {t("chat.report.success.message")}
+          </p>
+          <button
+            onClick={handleClose}
+            className="w-full rounded-xl bg-primary py-3 font-medium text-primary-foreground transition-colors hover:bg-primary/90"
+          >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)